src/main/java/org/elasticsearch/index/query/CommonTermsQueryBuilder.java

```diff
@@ -209,6 +281,9 @@ public void doXContent(XContentBuilder builder, Params params) throws IOExceptio
             builder.field("_name", queryName);
         }
 
+        // NO COMMIT: note that here we are always building the field disableCoord, highFreqOperator, lowFreqOperator, cutoffFrequency, 
+        // lowFreqMinimumShouldMatch, highFreqMinimumShouldMatch because of the assigned default value
```

Comment:
seems like as with boost, you should only print them out if they differ from the default?